src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -7,6 +7,7 @@ method java.lang.CharSequence length
 method java.lang.Comparable compareTo java.lang.Object
 method java.lang.Object getClass
 method java.lang.Object equals java.lang.Object
+method java.lang.Object getClass
```

Comment:
@jglick I tried to be nasty with `Class` but failed to do anything bad. As you can see in the following stack trace, Sandbox is preventing use of reflexion:

```
org.jenkinsci.plugins.scriptsecurity.sandbox.RejectedAccessException: Scripts not permitted to use method java.lang.Class getField java.lang.String
	at org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.StaticWhitelist.rejectMethod(StaticWhitelist.java:150)
```

I will remove the duplicated line